Source/Scene/Expression.js

```diff
@@ -144,6 +158,18 @@ define([
                 node.evaluate = node._evaluateEquals;
             } else if (node._value === '!==') {
                 node.evaluate = node._evaluateNotEquals;
+            } else if (node._value === '<') {
```

Comment:
1. So we should throw a syntax error when we know the types before runtime, but sometimes we need to evaluate the statement to determine the type. Should I throw the same kind of error at runtime for expressions that need to be evaluated?

2. How do we implement color `<`/`>`? Do we do it component wise and give red the highest precedence? This seems arbitrary and something rarely come up. `false < true` isn't really defined, unless you convert booleans to 0/1, so could we have `color('red') < color('green')` be undefined?